server/testsupport/timelineResponseTestDataBuilder.ts

```diff
@@ -0,0 +1,55 @@
+import type { TimelineResponse } from 'educationAndWorkPlanApiClient'
+
+export default function aValidTimelineResponse(): TimelineResponse {
+  return {
+    problemRetrievingData: false,
```

Comment:
This is not _quite_ right. A `TimelineResponse` object is the API object, straight out of the REST API. It does not have a `problemRetrievingData` field. It's only our view models that have this field, not the REST API types.
Just nuke this line (line 5) and it'll be OK 👍